qiskit_experiments/calibration/calibrations.py

```diff
@@ -0,0 +1,698 @@
+                if key in self._parameter_map:
+                    param = self._parameter_map[key]
+                elif (key.schedule, key.parameter, None) in self._parameter_map:
+                    param = self._parameter_map[(key.schedule, key.parameter, None)]
+                else:
+                    raise CalibrationError(
+                        f"Ill configured calibrations {key} is not present and has not default value."
+                    )
+
+                if param not in binding_dict:
+                    binding_dict[param] = self.get_parameter_value(
+                        key.parameter, key.qubits, key.schedule, group=group
+                    )
+
+        return schedule.assign_parameters(binding_dict, inplace=False)
+
+    @staticmethod
+    def get_parameter_keys(
+        schedule: Schedule, keys: Set, binding_dict: Dict[Parameter, int], qubits: Tuple[int, ...]
+    ):
+        """
+        Recursive function to extract parameter keys from a schedule. The recursive
+        behaviour is needed to handle Call instructions. Each time a Call is found
+        get_parameter_keys is call on the subroutine of the Call instruction and the
+        qubits that are in the subroutine. This also implies carefully extracting the
+        qubits from the subroutine and in the appropriate order.
+
+        Args:
+            schedule: A schedule from which to extract parameters.
+            keys: A set of keys that will be populated.
+            binding_dict: A binding dictionary intended only for channels. This is needed
+                because calling assign_parameters on a schedule with a Call instruction will
+                not assign the parameters in the subroutine of the Call instruction.
+            qubits: The qubits for which we want to have the schedule.
+
+        Returns:
+            keys: The set of keys populated with schedule name, parameter name, qubits.
+        """
+
+        # schedule.channels may give the qubits in any order. This order matters. For example,
+        # the parameter ('cr', 'amp', (2, 3)) is not the same as ('cr', 'amp', (3, 2)).
+        # Furthermore, as we call subroutines the list of qubits involved might shrink. For
+        # example, part of a cross-resonance schedule might involve.
+        #
+        # pulse.call(xp)
+        # ...
+        # pulse.play(GaussianSquare(...), ControlChannel(X))
+        #
+        # Here, the call instruction might, e.g., only involve qubit 2 while the play instruction
+        # will apply to qubits (2, 3).
+
+        qubit_set = set()
+        for chan in schedule.channels:
+            if isinstance(chan, DriveChannel):
+                qubit_set.add(chan.index)
+
+        qubits_ = tuple(qubit for qubit in qubits if qubit in qubit_set)
+
+        for _, inst in schedule.instructions:
+
+            if isinstance(inst, Play):
+                for params in inst.pulse.parameters.values():
```

Comment:
`is_parametrized()` is supported for all instructions.

Reply:
05d61eb probably dealt with this comment.